presto-jdbc/src/main/java/io/prestosql/jdbc/PrestoDriverUri.java

```diff
@@ -81,6 +82,7 @@
     private String schema;
 
     private final boolean useSecureConnection;
+    private final String clientTags;
```

Comment:
We don't need a field for this. Instead, use `CLIENT_TAGS` directly in the getter method.